CodeEdit/Features/Tasks/TaskManager.swift

```diff
@@ -16,6 +16,7 @@ class TaskManager: ObservableObject {
     @ObservedObject var workspaceSettings: CEWorkspaceSettingsData
 
     private var settingsListener: AnyCancellable?
+    private var taskStatusListener: AnyCancellable?
```

Comment:
```suggestion
```

Is this used?

Reply:
No, it left over from a test. I meant to remove it

Reply:
Removed